ngraph/core/reference/include/ngraph/runtime/reference/region_yolo.hpp

```diff
@@ -0,0 +1,175 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include <cmath>
+
+#include "ngraph/shape.hpp"
+
+namespace ngraph
+{
+    namespace runtime
+    {
+        namespace reference
+        {
+            static inline int entry_index(int width,
```

Comment:
I understand you`re introducing this function so you don't have to use CoordinateTransform? Did you check that is positively impacts the performance?

Reply:
Good point. In this case I did not make such comparison for two reasons.

- There was no reference implementation before, so it would involve to make the reference implementation twice (with and without CoordinateTransform)

- This operator is also not part of Constant Folding optimization pass, so that it was not included in the scope of reviewing performance via vtune